src/chrome/content/rules/GoNewSoft.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="GoNewSoft.com">
+        <target host="gonewsoft.com" />
+        <target host="www.gonewsoft.com" />
+		
+		<rule from="^http://(?:www\.)?gonewsoft\.com/" to="https://gonewsoft.com/"/>
```

Comment:
Yes. But with the simple rule I suggested, visitors will go first to https://www.gonewsoft.com/, and then be redirected to https://gonewsoft.com/ by the server. We tend to prefer that style, unless the www version doesn't have a certificate.